Nodejs/Product/Nodejs/Options/NodejsGeneralOptionsControl.cs

```diff
@@ -66,6 +66,11 @@ public partial class NodejsGeneralOptionsControl : UserControl {
             _waitOnNormalExit.Checked = page.WaitOnNormalExit;
             _editAndContinue.Checked = page.EditAndContinue;
             _checkForLongPaths.Checked = page.CheckForLongPaths;
+            _webkitDebugger.Checked = page.UseWebKitDebugger;
+
+#if !DEV15
+            _webkitDebugger.Enabled = false;
```

Comment:
it appears but is disabled. I can hide it, but then there is a weird gap between the normal options and the survey options.